docs/datasets.rst

```diff
@@ -0,0 +1,37 @@
+datasets package
```

Comment:
Very nice! Was this `.rst` automatically generated or did you write it?

If you wrote it, let's try to use the same syntax/headings than `geometry.rst` [here](https://raw.githubusercontent.com/geomstats/geomstats/master/docs/geometry.rst).

For example, writing:
```
********
Datasets
********
```

Reply:
I generated it by `sphinx-apidoc` command.